app/controllers/calls_controller.rb

```diff
@@ -0,0 +1,78 @@
+class CallsController < ApplicationController
+
+  after_filter :set_header
+
+  # Public: initiates the call process via a request from twillio
+  #
+  # CallSid - default param from twilio (required)
+  def start
+    response = Twilio::TwiML::Response.new do |r|
+      r.Say 'We need you to connect with your congressperson and senator. We are going to put you in touch with '
+      r.Redirect calls_new_connection_url, method: 'get'
+    end
+
+    render_twiml response
+  end
+
+  # Public: initiates a new outgoing call to congress
+  #
+  # CallSid - default param from twilio (required)
+  def new_connection
+    response = Twilio::TwiML::Response.new do |r|
+      if active_call.target_legislators.any?
+        connection = active_call.create_connection!
+        r.Say 'We will connect you in just a moment. Press star at any time to disconnect from your legislator.'
+        target_number = ENV['FAKE_CONGRESS_NUMBER'] || connection.legislator.phone
+        r.Dial target_number, 'action' => calls_connection_gather_prompt_url, 'hangupOnStar' => true
```

Comment:
Line is too long. [100/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.